CLAUDE.md

```diff
@@ -6,7 +6,7 @@ This file provides guidance to Claude Code (claude.ai/code) when working with co
 
 ### Primary Documentation (read these first)
 - **[README.md](README.md)** - Repository overview, package descriptions, all doc links
-- **[CONTRIBUTING.md](CONTRIBUTING.md)** - Setup, testing workflow, yarn linking
+- **[CONTRIBUTING.md](CONTRIBUTING.md)** - Setup, testing workflow, pnpm linking
```

Comment:
```suggestion
- **[CONTRIBUTING.md](CONTRIBUTING.md)** - Setup, testing workflow, package linking
```

Not a blocker, but explicitly tying to the package manager in use doesn't give much value but maintenance effort.